src/main/java/org/broadinstitute/consent/http/models/DataAccessRequest.java

```diff
@@ -24,6 +25,9 @@
 @JsonInclude(Include.NON_NULL)
 public class DataAccessRequest {
 
+  public static final long EXPIRATION_DURATION_MILLIS = 365L * 24 * 60 * 60
+      * 1000; // 365 days/year 24 hours/day 60 minutes/hour 60 seconds/minute 1000 ms/second
```

Comment:
use `TimeUnit` to avoid math
```suggestion
  public static final long EXPIRATION_DURATION_MILLIS = TimeUnit.DAYS.toMillis(365L);
```